api/drop_chunks.md

```diff
@@ -33,14 +33,16 @@ based on a hash partition.
 |Name|Type|Description|
 |-|-|-|
 |`relation`|REGCLASS|Hypertable or continuous aggregate from which to drop chunks.|
-|`older_than`|INTERVAL|Specification of cut-off point where any full chunks older than this timestamp should be removed.|
 
 ## Optional arguments
 
 |Name|Type|Description|
 |-|-|-|
-|`newer_than`|INTERVAL|Specification of cut-off point where any full chunks newer than this timestamp should be removed.|
+|`older_than`|ANY|Specification of cut-off point where any full chunks older than this timestamp should be removed.|
+|`newer_than`|ANY|Specification of cut-off point where any full chunks newer than this timestamp should be removed.|
```

Comment:
```suggestion
|`older_than`|ANY|Chunks older than this timestamp are removed.|
|`newer_than`|ANY|Chunks newer than this timestamp are removed.|
```

Reply:
It's not straight forward like that. If both `older_than` and `newer_than` are specified then the chunks which lie in the range between these two values are considered.

Reply:
Well, the before and after sentences say the same thing, the update just in a more direct way.